app/views/shared/_footer.html.erb

```diff
@@ -60,23 +60,23 @@
 
         <ol class="unstyled-list l-footer-social__list">
           <li class="l-footer-social__list-item twitter">
-            <a href="http://twitter.com/YourMoneyAdvice" title="Your Money Advice on Twitter">
+            <a href="http://twitter.com/YourMoneyAdvice" title="Your Money Advice on Twitter" class="footer-social__link">
```

Comment:
Wondered about that but thought the l- prefix implied a style specific to layout which this isn't. Happy to change that though.